ecc/scripts/esp-controller.js

```diff
@@ -275,14 +296,13 @@ export async function removeSpeakerFromEvent(speakerId, eventId) {
 }
 
 export async function updateSpeaker(profile, seriesId) {
+  const nSpeaker = convertToNSpeaker(profile);
   const { host } = getAPIConfig().esp[ECC_ENV];
-  const nProfile = { ...profile, photo: undefined };
-  const raw = JSON.stringify({ ...nProfile, seriesId });
+  const raw = JSON.stringify({ ...nSpeaker, seriesId });
   const options = await constructRequestOptions('PUT', raw);
 
   const resp = await fetch(`${host}/v1/series/${seriesId}/speakers/${profile.speakerId}`, options)
-    .then((res) => res.json())
-    .catch((error) => window.lana?.log(`Failed to update speaker. Error: ${error}`));
```

Comment:
Any reason we removed the lana log here?

Reply:
unintentional, thanks for catching this.